db/indexes.go

```diff
@@ -360,9 +372,10 @@ func waitForIndexes(bucket *base.CouchbaseBucketGoCB, useXattrs bool) error {
 func waitForIndex(bucket *base.CouchbaseBucketGoCB, indexName string, queryStatement string) error {
 
 	for {
-		_, err := bucket.Query(queryStatement, nil, gocb.RequestPlus, true)
+		r, err := bucket.Query(queryStatement, nil, gocb.RequestPlus, true)
 		// Retry on timeout error, otherwise return
 		if err == nil {
+			_ = r.Close()
```

Comment:
I think this might've been a left over test from when I was running into file descriptor exhaustion. Think I can safely undo this without any consequences.